src/containers/RecentData.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types"
+import Data from "../components/data"
+import statsClient from "panoptes-client/lib/stats-client";
+import { config } from "../config"
+
+class RecentData extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      totalTableClassifications: 0,
+      totalProjectClassifications: 0
+    };
+  }
+
+  componentDidMount() {
+    this.setTotalCount();
+    this.settotalTableClassifications();
+  }
+
+  setTotalCount() {
+    statsClient
+      .query({
+        projectID: config.projectID,
+        period: "year",
+        type: "classification"
+      })
+      .then((data) => {
+        let totalProjectClassifications = 0;
+        data.map(year => totalProjectClassifications += year.doc_count);
+        this.setState({ totalProjectClassifications });
+      })
+  }
+
+  settotalTableClassifications() {
+    statsClient
+      .query({
+        workflowID: config.tableWorkflowID,
+        period: "day",
```

Comment:
This is still set up to retrieve daily classification counts, so I'll have to adjust this in a new commit.